docs/api.md

```diff
@@ -815,19 +821,20 @@ Shortcut for [page.mainFrame().title()](#frametitle).
 #### page.tracing
 - returns: <[Tracing]>
 
-#### page.type(text, options)
+#### page.type([selector, ]text[, options])
```

Comment:
let's make selector unconditional here and in `page.press`. This would align better with the rest of the API. Having selector optional here might create false expectations from other API methods.

We'll break all the beta users, but that's why we're unstable yet.

Reply:
How do you feel above removing `page.press`

Reply:
> How do you feel above removing page.press

@JoelEinbinder i like it.